### DISCUSSION TITLE
Inaccurate Website Status Alerts with Cloudflare DNS

### DISCUSSION THREAD

**Discussion by goncalopolido** [upptime/upptime#869] [Q&A]
All of my websites are incorrectly labeled as "down" even though they are operational. I'm currently utilizing Cloudflare DNS services. You can verify the status of one of my websites at [status.polido.pt](https://status.polido.pt/).

upptime repo: https://github.com/7zx/status.polido.pt

**Comment by AnandChowdhary** [comment]
I can see from the logs that the site is responding with an HTTP 403 (https://github.com/7zx/status.polido.pt/actions/runs/6577658471/job/17869744385#step:3:14). When I try `curl -I` on my device, I see that it indeed responds with HTTP/2 200, so you're right, most likely Cloudflare's bot protection.

**Reply by goncalopolido** [reply]
could be, since i have firewall rules for bots.

**Reply by AnandChowdhary** [reply]
Aah that's likely it
